workers/tests/integrationtests/test_indicator_ghs_pop_comparison_buildings.py

```diff
@@ -30,6 +31,10 @@ def test(self):
         self.assertIsNotNone(self.indicator.feature_count)
         self.assertIsNotNone(self.indicator.feature_count_per_sqkm)
         self.assertIsNotNone(self.indicator.pop_count_per_sqkm)
+        self.assertIsNotNone(self.indicator.result.timestamp_osm)
+        self.assertIsInstance(self.indicator.result.timestamp_osm, datetime)
+        self.assertIsNotNone(self.indicator.result.timestamp_oqt)
+        self.assertIsInstance(self.indicator.result.timestamp_oqt, datetime)
```

Comment:
@matthiasschaub do you see a reason why we need the additional `self.assertIsNotNone` here? I think these checks are redundant. The same applies to the changes in the other tests.